LayoutTests/svg/custom/external-paintserver-reference-expected.svg

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" standalone="no"?>
+<html xmlns="http://www.w3.org/1999/xhtml" style="height:100%"/>
```

Comment:
I don't understand how this can be a valid reference given that `LayoutTests/svg/custom/external-paintserver-reference-expected.txt` lists a variety of colors that are no longer mentioned here.